### DISCUSSION TITLE
Unable to connect snowflake with user/password authentication

### DISCUSSION THREAD

**Discussion by anikarunakaran** [Snowflake-Labs/terraform-provider-snowflake#937] [Q&A]
Does anyone know why i am getting below given error while using user/password authentication for snowflake?.

terraform {
  required_providers {
    snowflake = {
      source  = "chanzuckerberg/snowflake"
      version = "0.28.7"
    }
  }
}

provider "snowflake" {

  # Configuration options
  username = "admin"
  account  = "xxxxxx"
  region   = "east-us-2.azure"
  password = "xxxxxxxx"
}

resource "snowflake_database" "demo_db" {
  name = "demo_db"
}

=====================================

 Error: Conflicting configuration arguments
│
│   with provider["registry.terraform.io/chanzuckerberg/snowflake"],
│   on main.tf line 10, in provider "snowflake":
│   10: provider "snowflake" {
│
│ "private_key_path": conflicts with password

**Comment by danu165** [comment]
Check if you have `SNOWFLAKE_PRIVATE_KEY_PATH` in your env variables. That variable is injected into private_key_path
